<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonOpenFile">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flightbooking</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rentalcar Reservation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDontModify3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hotelbooking</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelId">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditId">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrice">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrice">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFromDestination">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>From Destination:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelToDestination">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To Destination:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAirline">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Airline</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFromDestination">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditToDestination">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAirline">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlightArrival">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrival</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlightDeparture">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Departure</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFlightArrival">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditFlightDeparture">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditHotelTo">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>340</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditHotelFrom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hotel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelTo">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>340</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelFrom">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHotelTown">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Town:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHotelName">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHotelTown">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCarCompany">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCarPickupLocation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCarReturnLocation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>391</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditCarFrom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditCarTo">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>480</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarCompany">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Company</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarPickupLocation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pickup Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarReturnLocation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Return Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarFrom">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarTo">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCustomerId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearchCustomer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTravels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Travel-Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End date</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetBookings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Travel-Id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTravelId">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <addaction name="menua"/>
  </widget>
  <action name="actionSearch_with_ID">
   <property name="text">
    <string>Search with ID</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
